api/integrations/codebase/zulip_codebase_mirror

```diff
@@ -77,7 +77,7 @@ def make_api_call(path):
     # type: (str) -> Optional[List[Dict[str, Any]]]
     response = requests.get("https://api3.codebasehq.com/%s" % (path,),
                             auth=(config.CODEBASE_API_USERNAME, config.CODEBASE_API_KEY),
-                            params={'raw': True},
+                            params={'raw': 'True'},
```

Comment:
Hmm, I think this may be a case where we should fix typeshed to annotate `requests.get` properly; I think in this case the type on our code was probably correct.